x-pack/plugins/infra/public/pages/metrics/inventory_view/components/node_details/tabs/logs.tsx

```diff
@@ -6,7 +6,7 @@
  */
 
 import React, { useCallback, useMemo, useState } from 'react';
-import { useThrottle } from 'react-use';
+import useThrottle from 'react-use/lib/useThrottle';
```

Comment:
This PR fixes an eslint rule that should have been firing, but wasn't. I've updated this import to comply with the fixed rule.